### PR TITLE
[OSCD] Added a rule to detect execution of runonce with suspicious parameters

### PR DESCRIPTION
**Pull request by V3T0**
Added a rule to detect the execution of runonce with suspicious parameters. If runonce has been executed with /AlternateShellStartup param, it'll run payload configured in HKLM\SOFTWARE\Microsoft\Active Setup\Installed Components and StubPath value

### CODE REVIEW

File: rules/windows/process_creation/win_susp_runonce_execution.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+title: Run Once Task Execution as Configured in Registry
+id: 198effb6-6c98-4d0c-9ea3-451fa143c45c
+description: This rule detects the execution of Run Once task as configured in the registry
+author: Avneet Singh '@v3t0_', oscd.community
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_susp_runonce_execution.yml:4]: could you please apply [this](https://github.com/Neo23x0/sigma/pull/1210/files#r506965174) recommendation here as well?

File: rules/windows/process_creation/win_susp_tracker_execution.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+title: DLL Injection with Tracker.exe
+id: 148431ce-4b70-403d-8525-fcc2993f29ea
+description: This rule detects DLL injection and execution via LOLBAS - Tracker.exe
+author: Avneet Singh '@v3t0_', oscd.community
```
Comment (yugoslavskiy) [inline rules/windows/process_creation/win_susp_tracker_execution.yml:4]: could you please apply [this](https://github.com/Neo23x0/sigma/pull/1210/files#r506965174) recommendation here as well?

File: rules/windows/registry_event/sysmon_runonce_persistence.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+title: Run Once Task Configuration in Registry
+id: c74d7efc-8826-45d9-b8bb-f04fac9e4eff
+description: Rule to detect the configuration of Run Once registry key. Configured payload can be run by runonce.exe /AlternateShellStartup
+author: Avneet Singh '@v3t0_', oscd.community
```
Comment (yugoslavskiy) [inline rules/windows/registry_event/sysmon_runonce_persistence.yml:4]: could you please apply [this](https://github.com/Neo23x0/sigma/pull/1210/files#r506965174) recommendation here as well?

**Comment by aw350m33d** [comment]
The same request as in https://github.com/Neo23x0/sigma/pull/1215#issuecomment-711086584. 🙏🏻

**Comment by V3T0** [comment]
Closing this faulty PR as it contains other commits. 
Opened a new PR: #1233

**Pull request closed without merging**